engine/platform/sdl/vid_sdl.c

```diff
@@ -812,8 +812,13 @@ qboolean VID_CreateWindow( int width, int height, window_mode_t window_mode )
 	{
 		if( !glw_state.initialized )
 		{
-			if( !GL_CreateContext( ))
-				return false;
+			while( !GL_CreateContext( ))
```

Comment:
I don't get it. There is already function `VID_CreateWindowWithSafeGL` that iterates over safegl options.

Why there is a second loop?